Sources/ReleaseServer/ReleaseServer.WebApi/Repositories/ReleaseArtifactRepository.cs

```diff
@@ -200,6 +199,53 @@ public List<string> GetPlatforms(string productName, string version)
 
             return platforms.OrderBy(p => p).ToList();
         }
+        
+        public BackupInformationModel RunBackup()
+        {
+            var timeStamp = DateTime.UtcNow.ToString("yyyy-MM-dd-HH-mm-ss");
```

Comment:
To provide an easier differentiation I would suggest to use an underscore between date and time:

```suggestion
            var timeStamp = DateTime.UtcNow.ToString("yyyy-MM-dd_HH-mm-ss");
```

Reply:
Realized, see:
https://github.com/Traeger-GmbH/release-server/blob/feature/implement-backup-mechanism/Sources/ReleaseServer/ReleaseServer.WebApi/Repositories/ReleaseArtifactRepository.cs#L210